apps/daimo-mobile/src/sync/sync.ts

```diff
@@ -24,7 +25,15 @@ import { Account, getAccountManager } from "../model/account";
 //   ...occasionally otherwise
 export function startSync() {
   console.log("[SYNC] APP LOAD, starting sync");
-  maybeSync(true);
+  maybeSync(true).then((success) => {
+    if (success) {
+      // create small delay to let interface render becuase this callback
+      // run right after getting data so the interface still got to adapt
+      setTimeout(() => {
+        SplashScreen.hideAsync();
+      }, 300);
+    }
+  });
```

Comment:
hmm we want to hide the splash screen as soon as possible

we could do it in `finally` here regardless of whether that initial sync succeeded

(we can update `useNetworkState()` so that the app shows offline as soon as initial sync fails. either here, or i'm happy to do it in a future PR.)

additionally: in `App` render > useEffect, hide the splashscreen as soon `account.lastBlockTimestamp` is less than 10 minutes ago. this means that it'll hide instantly once we're rendering with fresh data (and the setTimeout one here is just a failsafe)